projects/packages/videopress/src/client/admin/components/video-row/index.tsx

```diff
@@ -0,0 +1,263 @@
+import { Text, Button, useBreakpointMatch } from '@automattic/jetpack-components';
+import { Popover } from '@wordpress/components';
+import { dateI18n } from '@wordpress/date';
+import { sprintf, __ } from '@wordpress/i18n';
+import { Icon, image, trash, chevronDown, chevronUp } from '@wordpress/icons';
+import classNames from 'classnames';
+import { useState, useRef } from 'react';
+import privacy from './privacy-icon';
+import styles from './style.module.scss';
+
+const millisecondsToMinutesAndSeconds = ( milliseconds: number ) => {
+	const minutes = Math.floor( milliseconds / 60000 );
+	const seconds = Math.floor( ( milliseconds % 60000 ) / 1000 );
+	return `${ minutes }:${ seconds < 10 ? '0' : '' }${ seconds }`;
+};
```

Comment:
This seems like it should be an imported helper function.

Reply:
we have a few helpers defined in the Jetpack plugin if I'm not wrong. We can migrate them and update the code, but surely IMO in a follow-up one.

Reply:
Yes, let’s move at some moment 👍